rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/test_is_baseline_system_11_1.py

```diff
@@ -312,3 +444,27 @@ def test__is_baseline_system_11_1C__test_json_true():
         )
         == HVAC_SYS.SYS_11_1C
     )
+
+
+def test__is_baseline_system_unmatched1__true():
+    assert (
+        is_baseline_system_11_1(
+            SYS_11_1_TEST_UNMATCHED_RMD["ruleset_model_instances"][0],
+            "System Type Unmatched 1",
+            ["Air Terminal 1"],
+            ["Thermal Zone 1"],
+        )
+        == HVAC_SYS.UNMATCHED
+    )
+
+
+def test__is_baseline_system_unmatched2__true():
```

Comment:
Similarly, give a short description of the test logic here will be great.

Reply:
Good point! I just added the description.